### ISSUE TITLE
[Request] mobile version 

### ISSUE THREAD

**Issue by mat926** [elliotwaite/thumbnail-rating-bar-for-youtube#42]
I know this is probably out of scope of the extension. But could you or somebody please make a mobile version of this? For iOS it could be a Cydia tweak, for Android maybe try to get it on YouTube Vanced.

**Comment by elliotwaite** [comment]
Agreed. Porting something like this to mobile would be great. However, I currently don't know how that would be possible. But I'll leave this issue open in case others want to discuss it.

**Comment by OneSons** [comment]
@elliotwaite what about a version for m.youtube.com? That way you could use the same extension through something like Kiwi Browser or Firefox.

**Comment by elliotwaite** [comment]
@OneSons, good idea. I didn't know you could install extensions on mobile browsers, but it looks like you can. I'll try to get this extension to work for mobile browsers soon, but I'll probably wait till after figuring out the YouTube removing dislikes issue first (which is being discussed [here](https://github.com/elliotwaite/thumbnail-rating-bar-for-youtube/issues/50)).

**Comment by leodoi3** [comment]
Is there an api available so you could have this work in Youtube Vanced app?

**Comment by PandoraHayami** [comment]
Any updates on this issue? The Return YouTube Dislike extension do support m.youtube.com if it helps.

**Comment by PandoraHayami** [comment]
![Screenshot_20220212-111819](https://user-images.githubusercontent.com/79670265/153721255-6b51d62f-b628-4f78-89a0-386d4ef7dd9b.png)
Yep, it works.

**Comment by elliotwaite** [comment]
Please report anything that looks off in the mobile version. I have not tested it much yet.

For example, I just realized that it isn't currently working on YouTube's mobile gaming site (m.youtube.com/gaming), but this will be fixed in the next version (1.8.2), which I just submitted for review and should be released within a few hours.

**Comment by Jgr9** [comment]
Any chance this could come to Firefox Android?  It can do Extensions now.
